### PR TITLE
Arts 265

### PR DESCRIPTION
**Pull request by darrengsmithson**
remove web oauth2 security in trial-config-service

### CODE REVIEW

**Comment by darrengsmithson** [comment]
Ive left a 'hard-coded' userId as we removed the userId which we retrieved from the authentication object, this is needed to create the bootstrap user. Once we decide where to get the user from we can change..

File: trial-config-service/src/main/java/uk/ac/ox/ndph/mts/trial_config_service/config/WebConfig.java [hunk 1]
```diff
@@ -1,80 +1,17 @@
...
-    private String authServer;
-
-    private List<AuthorizationScope> authorizationScopeList;
-
-    WebConfig() {
-        authorizationScopeList = new ArrayList<>();
-    }
-
-    @Bean
-    public Docket api() {
```
Comment (shajidissa) [inline trial-config-service/src/main/java/uk/ac/ox/ndph/mts/trial_config_service/config/WebConfig.java:49]: This is still required for swagger, please put it back in

**Review by shajidissa — CHANGES_REQUESTED** [review]
Get swagger back working

File: trial-config-service/src/main/java/uk/ac/ox/ndph/mts/trial_config_service/config/WebConfig.java (hunk 1 above)
Comment (darrengsmithson) [reply trial-config-service/src/main/java/uk/ac/ox/ndph/mts/trial_config_service/config/WebConfig.java:49]: Hi Saj - readded removed Swagger config

**Review by shajidissa — APPROVED** [review]

**Review by mdp0 — APPROVED** [review]

**Comment by shajidissa** [comment]
merged

**Pull request merged**